omero/developers/Web/CreateApp.rst

```diff
@@ -6,86 +6,93 @@ to get you familiar with the Django framework. The more you know about
+follow the install instructions under :doc:`Deployment`.
 
 If you want to make changes to the OMERO.web code itself, go to
 :doc:`/developers/Web/EditingOmeroWeb`.
 
-You can place your app anywhere on your :envvar:`PYTHONPATH`,
-as long as it can be imported by OMERO.web.
+Create an app from the template example
+---------------------------------------
 
-Creating an app
----------------
+Go to `https://github.com/will-moore/omero-web-apps-examples <https://github.com/will-moore/omero-web-apps-examples>`_.
+Click 'Use this template' as `described here
+<https://help.github.com/en/articles/creating-a-repository-from-a-template>`_
+and choose a name for your new app, for example ``omero-demo-webapp``.
 
-We suggest you use `GitHub <https://github.com/>`_ (as we do) since it is much easier for us to
-help you with any problems you have if we can see your code. The steps below
-describe how to create a stand-alone git repository for your app, similar to
-`omero-webtest <https://github.com/openmicroscopy/omero-webtest>`_.
-If you do not want to use GitHub, simply ignore the steps related to GitHub.
+Go to the directory where you want your app to live and clone it:
 
-The steps below describe how to set up a new app. You should choose an
-appropriate name for your app and use it in place of <organization-appname>:
+::
 
+    $ cd /path/to/dir/
+    $ git clone https://github.com/will-moore/omero-demo-webapp
 
-Add your app to your PYTHONPATH
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can now use `omero-web-docker <https://github.com/ome/omero-web-docker/>`_
+to run the app. Here we add ``minimal_webapp`` to the installed apps and map the
+directory to the ``site-packages`` directory in the Docker instance so that our
+app is on the :envvar:`PYTHONPATH`, described in more detail below.
+We also choose the OMERO server we want to connect to.
+You need to edit the ``demo.openmicroscopy.org`` and ``/path/to/dir/`` in the
+following command:
 
-Your app needs to be within a directory that is on your :envvar:`PYTHONPATH`.
-We usually create a new container for a new app, and add it to the
-:envvar:`PYTHONPATH`.
 ::
 
-    $ mkdir organization-appname
-    $ cd organization-appname
-    $ export PYTHONPATH=$PYTHONPATH:/path/to/organization-appname
+    $ docker run -it -e OMEROHOST=demo.openmicroscopy.org -p 4080:4080 -e CONFIG_omero_web_apps='["minimal_webapp"]' -v /path/to/dir/omero-demo-webapp/minimal-webapp/minimal_webapp:/opt/omero/web/venv/lib/python2.7/site-packages/minimal_webapp openmicroscopy/omero-web-standalone
 
+This will run Docker interactively in the terminal.
+Now go to `http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
+in your browser.
+You should be redirected to the login screen and then back to the ``minimal_webapp``
+page which will display your Name and list your Projects.
 
-OR you could simply choose an existing location:
+Adding your app to your PYTHONPATH
```

Comment:
This is the place where the flow breaks for me: What about the docker now I was working with up till this line ? How does it bind ?

Reply:
This explains some of the python stuff above. It's more explanation than walkthrough.

Reply:
I think adding a sentence explaining the threads here would be sufficient. Preferably, it could be rather ``above`` as you said, and stating something in the vein of ``we will make a short detour showing docker, which you might use for this, then, later, we come back to the OMERO.web server installed directly on the machine.`` Or maybe just ``the docker is  a detour (additional option) serving purpose of xxx``  would be sufficient.